helm/dashboards/charts/public_dashboards/dashboards/shared/public/servicemonitors-overview.json

```diff
@@ -275,19 +281,20 @@
           "sortDesc": true
         },
         "tooltip": {
+          "hideZeros": false,
           "mode": "single",
           "sort": "none"
         }
       },
+      "pluginVersion": "12.1.0",
       "targets": [
         {
           "datasource": {
             "type": "prometheus",
             "uid": "${datasource}"
           },
           "editorMode": "code",
-          "expr": "sum by (cluster_id,pod) (container_memory_usage_bytes{cluster_id=\"$cluster\", pod=~\".*prometheus-agent.*\"})",
-          "expr": "sum by (cluster_id,pod) (container_memory_rss{cluster_id=\"$cluster\", pod=~\".*prometheus-agent.*\", container=~\".+\"})",
+          "expr": "sum by (cluster_id,pod) (container_memory_rss{cluster_id=\"$cluster\", pod=~\".*alloy-metrics.*\", container=~\".+\"})",
```

Comment:
We still run some prometheus agents though right? Can we not support both and add a comment?

Reply:
I have no clue, I never use it, but we do run it on 10+ clusters still